core-plugins/docs/Window-windower.md

```diff
@@ -0,0 +1,59 @@
+# Window Plugin
+
+
+Description
+-----------
+The Window plugin is used to window a part of a streaming pipeline.
+
+
+Use Case
+--------
+Windows are used when you wish to group smaller batches of events into larger batches of events.
+For example, if your streaming pipeline has a batch interval of one second, you may want to accumulate
```

Comment:
batch interval is set on a pipeline level. It's like 'instances' for cdap-etl-realtime or 'schedule' for cdap-data-pipeline. I still need to add docs for cdap-data-streams.